command/v7/set_label_command_test.go

```diff
@@ -30,6 +32,21 @@ var _ = Describe("set-label command", func() {
 		executeErr error
 	)
 
+	checkBuildpackArgFunc := func() {
```

Comment:
I can see we're trying to remove duplication in the test here. I don't have a strong opinion either way.

Perhaps we could rename this function to make it clearer that we are expecting an error?

Reply:
Sure. I see that the upcoming annotation stories roll all five resource types into one story. I'm not big on lots of DRYing in tests, but it's likely to happen. Let me know if you prefer everything unrolled.

Reply:
Rename was done, and the same function was added to `command/v7/labels_command_test.go`